### DISCUSSION TITLE
Downloaded Files

### DISCUSSION THREAD

**Discussion by MizuWatchesAnime** [KRTirtho/spotube#58] [Q&A]
Where are the downloaded files saved?

**Comment by KRTirtho** [comment]
In the Downloads folder of your respective Operating System

**Reply by simanchal-45** [reply]
Can you add feature to change download location, like if I want save a song in sd card?

**Reply by KRTirtho** [reply]
Yeah, this can be useful

**Reply by nepalivai** [reply]
@KRTirtho seems like this issue have been fixed as now we can select where the file saves.
<img width="772" alt="image" src="https://github.com/KRTirtho/spotube/assets/108126089/b859d782-203b-49aa-b096-3e9e5e5d4908">
